### ISSUE TITLE
Can not be used in nextjs

### ISSUE THREAD

**Issue by pascalmtts** [murageh/save-progress#5]
**Describe the bug**
Unfortunately, the package cannot be used with Next.js. The following error message is displayed:
```
ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and '/Users/pascal/Development/filu/frontend/node_modules/@crispice/save-progress/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
```
**To Reproduce**
1. Try to import with `import { useSaveProgress } from '@crispice/save-progress';`
2. Use it in component `useSaveProgress({ key: 'test' });`
3. Following error is displayed:

![image](https://user-images.githubusercontent.com/12157131/220585113-4725c354-0093-4656-a34c-e4f3a5f25e49.png)

**Expected behavior**
Should work also with nextjs

Note: Maybe it is enough to remove the paragraph "type: 'module'" from package.json to make it work.

**Desktop (please complete the following information):**
 - OS: macOS 13.1
 - Browser: Chrome
 - Version 110.0.5481.100

**Comment by murageh** [comment]
Thanks for raising this issue @pascalmtts . I had not designed the hook with Next.js in mind, but the new version should handle that. Fixed in #6. Version `1.0.9` should work fine.

Let me know of any hitches. Beware, too, that the hook also depends on `formik` since the `AutoSaveForm` component uses it.

**Comment by pascalmtts** [comment]
Thanks for your quick response! Sounds good, I already use Formik so that should work fine then. Looking forward to try the new version!
